usagov/usagov-2021
# USAGOV-2019-Missing-Breadcrumbs-on-Wizard-Pages

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2019

## Description

Adjust wizard breadcrumb data to use existing breadcrumb template

## Testing Instructions


- Visit localhost/where-report-scams
- Confirm that the breadcrumbs are correctly showing the expected items:
  - `Home > Scams and fraud > Learn where to report a scam`
- Navigate through the steps of the wizard
- Confirm that the breadcrumbs show the same items

- Visit localhost/es/donde-reportar-una-estafa
- Confirm that the breadcrumbs are correctly showing the expected items:
  - `Página principal > Estafas y fraudes > Aprenda dónde reportar o denunciar una estafa`
- Navigate through the steps of the wizard
- Confirm that the breadcrumbs show the same items



## Type of Changes

  - Add screenshot showing what it should look like

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
